File: js/programmers/level.1/prgm_12930_1.js
function solution(s) {
  arr = s.split(" ");
  answer = "";

  for (let word of arr) {
    for (let i = 0; i < word.length; i++) {
      i % 2
        ? (answer += word[i].toLowerCase())
        : (answer += word[i].toUpperCase());
    }
    answer += " ";
  }

  return answer.slice(0, -1);
}

File: python/programmers/level.1/prgm_12930_1.py
def solution(s):
    arr = s.split(" ")
    answer = ""

    for word in arr:
        for i in range(len(word)):
            if i % 2 == 0:
                answer += word[i].upper()
            else:
                answer += word[i].lower()
        answer += " "

    return answer[:-1]